packages/plugin-client-drizzle/test/integration.ava.ts

```diff
@@ -0,0 +1,2187 @@
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John', verified: true });
+  const result = await db.select().from(usersTable);
+
+  t.deepEqual(result, [{ id: 1, name: 'John', verified: true, jsonb: null, createdAt: result[0]!.createdAt }]);
+});
+
+test.skip('insert many', async (t) => {
+  const { db } = t.context;
+
+  await db
+    .insert(usersTable)
+    .values([
+      { name: 'John' },
+      { name: 'Bruce', jsonb: ['foo', 'bar'] },
+      { name: 'Jane' },
+      { name: 'Austin', verified: true }
+    ]);
+  const result = await db
+    .select({
+      id: usersTable.id,
+      name: usersTable.name,
+      jsonb: usersTable.jsonb,
+      verified: usersTable.verified
+    })
+    .from(usersTable);
+
+  t.deepEqual(result, [
+    { id: 1, name: 'John', jsonb: null, verified: false },
+    { id: 2, name: 'Bruce', jsonb: ['foo', 'bar'], verified: false },
+    { id: 3, name: 'Jane', jsonb: null, verified: false },
+    { id: 4, name: 'Austin', jsonb: null, verified: true }
+  ]);
+});
+
+test.skip('insert many with returning', async (t) => {
+  const { db } = t.context;
+
+  const result = await db
+    .insert(usersTable)
+    .values([
+      { name: 'John' },
+      { name: 'Bruce', jsonb: ['foo', 'bar'] },
+      { name: 'Jane' },
+      { name: 'Austin', verified: true }
+    ])
+    .returning({
+      id: usersTable.id,
+      name: usersTable.name,
+      jsonb: usersTable.jsonb,
+      verified: usersTable.verified
+    });
+
+  t.deepEqual(result, [
+    { id: 1, name: 'John', jsonb: null, verified: false },
+    { id: 2, name: 'Bruce', jsonb: ['foo', 'bar'], verified: false },
+    { id: 3, name: 'Jane', jsonb: null, verified: false },
+    { id: 4, name: 'Austin', jsonb: null, verified: true }
+  ]);
+});
+
+test.skip('select with group by as field', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values([{ name: 'John' }, { name: 'Jane' }, { name: 'Jane' }]);
+
+  const result = await db.select({ name: usersTable.name }).from(usersTable).groupBy(usersTable.name);
+
+  t.deepEqual(result, [{ name: 'Jane' }, { name: 'John' }]);
+});
+
+test.skip('select with group by as sql', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values([{ name: 'John' }, { name: 'Jane' }, { name: 'Jane' }]);
+
+  const result = await db
+    .select({ name: usersTable.name })
+    .from(usersTable)
+    .groupBy(sql`${usersTable.name}`);
+
+  t.deepEqual(result, [{ name: 'Jane' }, { name: 'John' }]);
+});
+
+test.skip('select with group by as sql + column', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values([{ name: 'John' }, { name: 'Jane' }, { name: 'Jane' }]);
+
+  const result = await db
+    .select({ name: usersTable.name })
+    .from(usersTable)
+    .groupBy(sql`${usersTable.name}`, usersTable.id);
+
+  t.deepEqual(result, [{ name: 'Jane' }, { name: 'Jane' }, { name: 'John' }]);
```

Comment:
As the `id` is not numeric, the ordering can be different. Locally I get John, Jane, Jane. We get the right elements, but in a different order.